rmf_fleet_adapter_python/src/types/types.cpp

```diff
@@ -16,134 +33,178 @@
+
+    py::class_<Delivery>(m_type, "CPPDeliveryMsg")
+        .def(py::init(&make_delivery_msg),
+             py::arg("task_id") = "",
+             py::arg("pickup_place_name") = "",
+             py::arg("pickup_dispenser") = "",
+             py::arg("dropoff_place_name") = "",
+             py::arg("dropoff_ingestor") = "")
+        .def_property(
+            "task_id",
+            [&](Delivery &self) { return self.task_id; },
+            [&](Delivery &self,
+                std::string task_id) { self.task_id = task_id; })
+        .def_property(
+            "pickup_place_name",
+            [&](Delivery &self) { return self.pickup_place_name; },
+            [&](Delivery &self,
+                std::string pickup_place_name) { self.pickup_place_name = pickup_place_name; })
+        .def_property(
+            "pickup_dispenser",
+            [&](Delivery &self) { return self.pickup_dispenser; },
+            [&](Delivery &self,
+                std::string pickup_dispenser) { self.pickup_dispenser = pickup_dispenser; })
+        .def_property(
+            "dropoff_place_name",
+            [&](Delivery &self) { return self.dropoff_place_name; },
+            [&](Delivery &self,
+                std::string dropoff_place_name) { self.dropoff_place_name = dropoff_place_name; })
+        .def_property(
+            "dropoff_ingestor",
+            [&](Delivery &self) { return self.dropoff_ingestor; },
+            [&](Delivery &self,
+                std::string dropoff_ingestor) { self.dropoff_ingestor = dropoff_ingestor; });
+
+    py::class_<Loop>(m_type, "CPPLoopMsg")
+        .def(py::init(&make_loop_msg),
+             py::arg("task_id") = "",
+             py::arg("robot_type") = "",
+             py::arg("num_loops") = "",
+             py::arg("start_name") = "",
+             py::arg("finish_name") = "")
+        .def_property(
+            "task_id",
+            [&](Loop &self) { return self.task_id; },
+            [&](Loop &self,
+                std::string task_id) { self.task_id = task_id; })
+        .def_property(
+            "robot_type",
+            [&](Loop &self) { return self.robot_type; },
+            [&](Loop &self,
+                std::string robot_type) { self.robot_type = robot_type; })
+        .def_property(
+            "num_loops",
+            [&](Loop &self) { return self.num_loops; },
+            [&](Loop &self,
+                uint32_t num_loops) { self.num_loops = num_loops; })
+        .def_property(
+            "start_name",
+            [&](Loop &self) { return self.start_name; },
+            [&](Loop &self,
+                std::string start_name) { self.start_name = start_name; })
+        .def_property(
+            "finish_name",
+            [&](Loop &self) { return self.finish_name; },
+            [&](Loop &self,
+                std::string finish_name) { self.finish_name = finish_name; });
+
+    // TODO: Support More task_types
```

Comment:
We should also include `Clean` task here since the rest of the fleet adapter supports it.